docs/pages/api-reference/liveblocks-react-comments.mdx

```diff
@@ -526,11 +526,11 @@ The components displayed within the editor.
 
 <Table columns={["26%", "32%", "42%"]}>
 
-| Component            | Type                                                                    | Description                                                                                             |
-| -------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
-| `Mention`            | <code>ComponentType&lt;ComposerEditorMentionProps&gt;</code>            | The component used to display mentions.<br/>Defaults to the mention’s `userId` prefixed by an @.        |
-| `MentionSuggestions` | <code>ComponentType&lt;ComposerEditorMentionProps&gt;</code>            | The component used to display mention suggestions.<br/>Defaults to a list of the suggestions’ `userId`. |
-| `Link`               | <code>ComponentType&lt;ComposerEditorMentionSuggestionsProps&gt;</code> | The component used to display links.<br/>Defaults to the link’s `children` property.                    |
+| Component            | Type                                                         | Description                                                                                             |
+| -------------------- | ------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------- |
+| `Mention`            | <code>ComponentType&lt;ComposerEditorMentionProps&gt;</code> | The component used to display mentions.<br/>Defaults to the mention’s `userId` prefixed by an @.        |
+| `MentionSuggestions` | <code>ComponentType&lt;ComposerEditorMentionProps&gt;</code> | The component used to display mention suggestions.<br/>Defaults to a list of the suggestions’ `userId`. |
```

Comment:
```suggestion
| `MentionSuggestions` | <code>ComponentType&lt;ComposerEditorMentionSuggestionProps&gt;</code> | The component used to display mention suggestions.<br/>Defaults to a list of the suggestions’ `userId`. |
```

I think this was a typo

Reply:
Ah yep! I found the one below but missed this one. Thanks!